### PR TITLE
Add BHNT87

### PR DESCRIPTION
**Pull request by ligi**
Thanks @vog - used your script the first time - worked nice!

### CODE REVIEW

File: _posts/2023-03-28-no87-h4k.md
Comment (vog) [inline _posts/2023-03-28-no87-h4k.md:1]: LGTM

**Review by vog — COMMENTED** [review]
LGTM

**Review by lucasrangit — APPROVED** [review]

File: _posts/2023-03-28-no87-h4k.md
Comment (Hasona1991) [reply _posts/2023-03-28-no87-h4k.md:1]: > LGTM

Comment (Hasona1991) [reply _posts/2023-03-28-no87-h4k.md:1]: #304

Comment (Hasona1991) [reply _posts/2023-03-28-no87-h4k.md:1]: #304

Comment (Hasona1991) [reply _posts/2023-03-28-no87-h4k.md:1]: > [# 304](https://github.com/berlin-hack-and-tell/berlinhackandtell.rocks/issues/304)

**Pull request merged**